wordle-web/components/word-options.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-container fill-height justify-center>
+    <v-btn
+      color="red"
+      :disabled="wordleGame.gameOver"
+      @click=";[validWords(), showCountButton()]"
+    >
+      Fill ? word<br>
+      (Cheating)
+    </v-btn>
+    <v-dialog v-model="dialog" width="500">
```

Comment:
I too could not get the hint system to work.  When I click the button it always displays zero for me and when I click the zero, there are no words in the list to choose from.